pycbc/population/rates_functions.py

```diff
@@ -112,9 +111,9 @@ def log_rho_bg(trigs, bins, counts):
     '''
 
     trigs = np.atleast_1d(trigs)
-
+    
     N = sum(counts)
-
+    
     assert np.all(trigs >= np.min(bins)), 'triggers can not besmaller than bin lower limit'
```

Comment:
clarify error message : eg 'trigger SNR values cannot all be below the lowest bin limit'